hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestDataDirs.java

```diff
@@ -96,26 +91,4 @@ public void testDataDirParsing() throws Throwable {
     assertThat(locations.get(1).getStorageType(), is(StorageType.DISK));
     assertThat(locations.get(1).getUri(), is(dir1.toURI()));
   }
-
-  @Test(timeout = 30000)
-  public void testDataDirValidation() throws Throwable {
```

Comment:
Yes there's new tests in TestStorageLocationChecker added by HDFS-11119 which cover this better with different combinations of failed/healthy storages.